### PR TITLE
Polish and standardize Allies01

### LINKED ISSUE #19183: In the thick of it polish
Minor polish, but if you kill the three troops guarding the tech center before the two to the top left, one will kill Einstein when he is released.

The civilians have a random chance to be run over by the ore truck. It is possible for Tanya to C4 the ore truck as it unloads, but for a first mission, the timing and knowledge of C4ing vehicles seems unreasonable and this probably would look more like a bug for new players.

### PR DESCRIPTION
**Pull request by Smittytron**
This addresses and closes #19183, and standardizes player names. I used the translated .ini script to get things closer to what they were in the original.

### CODE REVIEW

**Comment by pchote** [comment]
For reference, this is the translated ini script:

<details><summary>Click to expand</summary>

```
ScriptedActor@Actor0: # tsla owned by house "USSR" at cell 71,59
	Sellable: True
ScriptedActor@Actor1: # powr owned by house "USSR" at cell 75,64
	Sellable: True
ScriptedActor@Actor2: # powr owned by house "USSR" at cell 67,57
	Sellable: True
ScriptedActor@Actor3: # powr owned by house "USSR" at cell 61,57
	Sellable: True
ScriptedActor@Actor4: # stek owned by house "USSR" at cell 61,60
	Trigger: los3
	Sellable: False
ScriptedActor@Actor5: # fact owned by house "USSR" at cell 69,62
	Sellable: False
ScriptedActor@Actor6: # dome owned by house "USSR" at cell 67,65
	Sellable: False
ScriptedActor@Actor7: # barr owned by house "USSR" at cell 61,64
	Sellable: False
ScriptedActor@Actor8: # tsla owned by house "USSR" at cell 67,67
	Sellable: False
ScriptedActor@Actor9: # tsla owned by house "USSR" at cell 60,66
	Sellable: False
ScriptedActor@Actor10: # weap owned by house "USSR" at cell 65,62
	Sellable: False
ScriptedActor@Actor11: # proc owned by house "USSR" at cell 73,58
	Sellable: False
ScriptedActor@Actor12: # kenn owned by house "USSR" at cell 64,65
	Sellable: True
ScriptedActor@Actor13: # powr owned by house "USSR" at cell 65,57
	Sellable: True
ScriptedActor@Actor14: # powr owned by house "USSR" at cell 77,64
	Sellable: True
ScriptedActor@Actor15: # powr owned by house "USSR" at cell 75,67
	Sellable: True
ScriptedActor@Actor16: # silo owned by house "USSR" at cell 59,64
	Sellable: True
ScriptedActor@Actor17: # powr owned by house "USSR" at cell 77,67
	Sellable: False
ScriptedActor@Actor19: # brl3 owned by house "USSR" at cell 60,57
	Sellable: True
ScriptedActor@Actor20: # barl owned by house "USSR" at cell 60,56
	Sellable: True
ScriptedActor@Actor21: # barl owned by house "USSR" at cell 61,56
	Trigger: det1
	Sellable: True
ScriptedActor@Actor22: # brl3 owned by house "USSR" at cell 60,58
	Sellable: True
ScriptedActor@Actor23: # barl owned by house "USSR" at cell 58,56
	Sellable: True
ScriptedActor@Actor24: # barl owned by house "USSR" at cell 59,59
	Sellable: True
ScriptedActor@Actor25: # jeep owned by house "Greece" at cell 63,50
	Action: Guard
ScriptedActor@Actor26: # harv owned by house "USSR" at cell 72,60
	Action: Harvest
ScriptedActor@Actor27: # jeep owned by house "Greece" at cell 62,50
	Action: Guard
ScriptedActor@Actor28: # jeep owned by house "Greece" at cell 64,50
	Action: Guard
ScriptedActor@Actor29: # dog owned by house "USSR" at cell 63,59
	Action: Hunt
	Trigger: dwig
ScriptedActor@Actor30: # e1 owned by house "USSR" at cell 64,58
	Action: Hunt
	Trigger: dwig
ScriptedActor@Actor31: # e1 owned by house "USSR" at cell 61,63
	Action: Sticky
	Trigger: eins
ScriptedActor@Actor32: # e1 owned by house "USSR" at cell 63,63
	Action: Sticky
	Trigger: eins
ScriptedActor@Actor33: # e2 owned by house "USSR" at cell 73,66
	Action: Guard
ScriptedActor@Actor34: # e1 owned by house "USSR" at cell 62,67
	Action: Guard
ScriptedActor@Actor35: # e1 owned by house "USSR" at cell 67,67
	Action: Sticky
ScriptedActor@Actor36: # e1 owned by house "USSR" at cell 65,67
	Action: Guard
ScriptedActor@Actor37: # e1 owned by house "USSR" at cell 56,60
	Action: Guard
ScriptedActor@Actor38: # e1 owned by house "USSR" at cell 62,55
	Action: Hunt
	Trigger: dwig
ScriptedActor@Actor39: # e1 owned by house "USSR" at cell 64,59
	Action: Hunt
	Trigger: dwig
ScriptedActor@Actor40: # e1 owned by house "USSR" at cell 64,61
	Action: Area Guard
ScriptedActor@Actor41: # e1 owned by house "USSR" at cell 58,60
	Action: Guard
ScriptedActor@Actor42: # e1 owned by house "Greece" at cell 64,49
	Action: Guard
ScriptedActor@Actor43: # e1 owned by house "Greece" at cell 63,49
	Action: Guard
ScriptedActor@Actor44: # e1 owned by house "Greece" at cell 62,49
	Action: Guard
ScriptedActor@Actor45: # c8 owned by house "England" at cell 74,50
	Action: Guard
ScriptedActor@Actor46: # c7 owned by house "England" at cell 76,48
	Action: Guard
ScriptedActor@Actor47: # e2 owned by house "USSR" at cell 62,56
	Action: Sticky
	Trigger: trp1
ScriptedActor@Actor48: # e2 owned by house "USSR" at cell 62,56
	Action: Sticky
ScriptedActor@Actor49: # e1 owned by house "Greece" at cell 64,49
	Action: Guard
ScriptedActor@Actor50: # e1 owned by house "Greece" at cell 62,49
	Action: Guard
CellTrigger@rnf1:
	Trigger: rnf1
	Cells: 62,55, 63,55, 64,55, 65,55, 66,55, 67,55, 68,55, 69,55, 70,55, 71,55, 72,55, 73,55, 74,55, 75,55, 76,55, 77,55, 78,55, 62,56, 63,56, 64,56, 65,56, 66,56, 67,56, 68,56, 69,56, 70,56, 71,56, 72,56, 73,56, 74,56, 75,56, 76,56, 77,56, 78,56
TeamType@sov1:
	Units: E1, E1, DOG
	House: 2
	Flags: IsPrebuilt, IsReinforcable
	RecruitPriority: 30
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint2" (53,79)
	Mission:
		0: Move to "waypoint3" (57,65)
		1: Guard area for 1f minutes
		2: Move to "waypoint2" (53,79)
TeamType@einst:
	Units: EINSTEIN
	House: 1
	Trigger: elos
	Flags: None
	RecruitPriority: 10
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint7" (62,61)
	Mission:
		0: Follow nearest friendly unit
TeamType@cruz:
	Units: CA
	House: 3
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint23" (66,80)
	Mission:
		0: Move to "waypoint23" (66,80)
		1: Move to "waypoint25" (64,75)
		2: Attack target type "Buildings (any)"
TeamType@heli:
	Units: TRAN
	House: 8
	Trigger: dcop
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint23" (66,80)
	Mission:
		0: Move to "waypoint23" (66,80)
		1: Move to "waypoint24" (53,49)
		2: Jump to line 2
TeamType@tanya:
	Units: E7, TRAN
	House: 1
	Trigger: tlos
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint10" (63,47)
	Mission:
		0: Move to "waypoint10" (63,47)
		1: Transports unload cargo / Minelayers lay mine
TeamType@rnf1:
	Units: E1, E1, JEEP, JEEP
	House: 1
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint11" (63,45)
	Mission:
		0: Move to "waypoint11" (63,45)
		1: Move to "waypoint10" (63,47)
TeamType@crz2:
	Units: CA
	House: 3
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint23" (66,80)
	Mission:
		0: Move to "waypoint23" (66,80)
		1: Move to "waypoint22" (72,75)
		2: Attack target type "Buildings (any)"
TeamType@crz3:
	Units: CA
	House: 3
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint23" (66,80)
	Mission:
		0: Move to "waypoint23" (66,80)
		1: Move to "waypoint27" (60,74)
		2: Attack target type "Buildings (any)"
TeamType@crz4:
	Units: CA
	House: 3
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint23" (66,80)
	Mission:
		0: Move to "waypoint23" (66,80)
		1: Move to "waypoint26" (68,76)
		2: Attack target type "Buildings (any)"
TeamType@rnf2:
	Units: JEEP, JEEP
	House: 1
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Origin: "waypoint11" (63,45)
	Mission:
		0: Move to "waypoint11" (63,45)
		1: Move to "waypoint10" (63,47)
TeamType@dwight:
	Units: C8
	House: 3
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Mission:
		0: Move to "waypoint5" (69,56)
TeamType@phil:
	Units: C7
	House: 3
	Flags: None
	RecruitPriority: 7
	InitNum: 0
	MaxAllowed: 0
	Mission:
		0: Move to "waypoint5" (69,56)
Trigger@elos:
	On:
		 Attached unit or building is destroyed
	Actions:
		 House "Greece" is defeated
		 Display text message 51 from tutorial.ini
	Disable: After running once
Trigger@set1:
	On:
		 0.0 minutes have elapsed
	Actions:
		 Reinforce with team "tanya"
		 Display text message 8 from tutorial.ini
	Disable: After running once
Trigger@eins:
	On:
		 Attached unit or building is destroyed
	Actions:
		 Reinforce with team "einst"
		 Run trigger "ein2"
	Disable: After running on all attached objects
Trigger@tlos:
	On:
		 Attached unit or building is destroyed
	Actions:
		 House "Greece" is defeated
		 Display text message 52 from tutorial.ini
	Disable: After running once
Trigger@revl:
	On:
		 0.1 minutes have elapsed
	Actions:
		 Reveal area around "waypoint20" (64,63)
		 Reveal area around "waypoint78" (70,60)
	Disable: After running once
Trigger@ein2:
	On:
		 Manual Trigger
	Actions:
		 Spawn flare with small vision radius at "waypoint0" (54,48)
		 Set global scenario variable 1
	Disable: After running once
Trigger@shel:
	On:
		 0.4 minutes have elapsed
		 AND
		 Global scenario variable 1 is set
	Actions:
		 Reinforce with team "cruz"
		 Reinforce with team "crz2"
	Disable: After running once
Trigger@ein3:
	On:
		 Global scenario variable 1 is set
	Actions:
		 Reinforce with team "heli"
		 Play speech message "FLAREN1"
	Disable: After running once
Trigger@los3:
	On:
		 Attached unit or building is destroyed
		 AND
		 Global scenario variable 1 is not set
	Actions:
		 House "Greece" is defeated
		 Display text message 62 from tutorial.ini
	Disable: After running once
Trigger@win:
	On:
		 Civilian unit owned by "1" left the map or loaded into helicopter.
	Actions:
		 House "Greece" is victorious
	Disable: After running once
Trigger@dwig:
	On:
		 Attached unit or building is destroyed
	Actions:
		 Create team "dwight"
		 Create team "dwight"
	Disable: After running on all attached objects
Trigger@shl2:
	On:
		 0.5 minutes have elapsed
		 AND
		 Global scenario variable 1 is set
	Actions:
		 Reinforce with team "crz3"
		 Reinforce with team "crz4"
	Disable: After running once
Trigger@rnf1:
	On:
		 Attached cell entered or attached building captured by house "Greece"
	Actions:
		 Reinforce with team "rnf2"
	Disable: After running once
Trigger@shl3:
	On:
		 0.3 minutes have elapsed
		 AND
		 Global scenario variable 1 is set
	Actions:
		 Play speech message "AARRIVE1_"
	Disable: After running once
Trigger@dcop:
	On:
		 Attached unit or building is destroyed
	Actions:
		 Play speech message "OBJNMET1"
		 House "Greece" is defeated
	Disable: After running once
Trigger@trp1:
	On:
		 Attached unit or building is destroyed
	Actions:
		 Run trigger "det1"
	Disable: After running once
Trigger@det1:
	On:
		 Manual Trigger
	Actions:
		 Destroy attached buildings
	Disable: After running once
Trigger@lowp:
	On:
		 House "USSR" is low power
	Actions:
		 Create team "phil"
	Disable: After running once
Trigger@ein4:
	On:
		 Global scenario variable 1 is set
	Actions:
		 Display text message 9 from tutorial.ini
	Disable: After running once
```
</details>

File: mods/ra/maps/allies-01/map.yaml [hunk 1]
```diff
@@ -16,24 +16,26 @@ Visibility: MissionSelector
 
 Categories: Campaign
 
+Translations:
+
```
Comment (abcdefg30) [inline mods/ra/maps/allies-01/map.yaml]: Please remove this again, see #19573.

**Comment by Smittytron** [comment]
Updated

**Review by abcdefg30 — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/9fc2fa9f8efc14e5de062861504017ca1ec13de9...1dfc0605c3aeade399d50568d2ae306c5e0ca0f0

**Pull request merged**